### PR TITLE
I18t added 

### PR DESCRIPTION
**Pull request by tarunsengar1987**

### CODE REVIEW

File: ni18n.config.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const supportedLngs = ['pt', 'en']
+
+export const ni18nConfig = {
+  fallbackLng: supportedLngs,
```
Comment (robdll) [inline ni18n.config.js]: this should be a string.
Use 'pt' as fallback

File: pages/_app.js [hunk 2]
```diff
@@ -1,16 +1,56 @@
...
-      <meta name="viewport"  key="viewport" content="initial-scale=1.0, width=device-width" />
-      <meta name="description" key="description" content="Descubra as mandalas e seus significados. Aprenda a desegnar, colorir e manifestar." />
-    </Head>
-       
-    <Component {...pageProps} />
-  </>
+  const { t, i18n } = useTranslation();
+
+  // useSyncLanguage(locale);
+
```
Comment (robdll) [inline pages/_app.js]: remove empty line and comment

File: pages/_app.js [hunk 3]
```diff
@@ -1,16 +1,56 @@
...
+  const changeLanguage = () => {
+    const locale = Cookies.get('MY_LANGUAGE');
+    const urlParams = new URLSearchParams(window.location.search);
+    const language = urlParams.get("lang");
+    const browserLang = navigator.language || navigator.userLanguage; 
+
+    if (language) {
```
Comment (robdll) [inline pages/_app.js:35]: this was supposed to use [detectors](https://github.com/i18next/i18next-browser-languageDetector)

File: pages/_app.js [hunk 4]
```diff
@@ -1,16 +1,56 @@
...
+      window.document.cookie = `MY_LANGUAGE=${language}`;
```
Comment (robdll) [inline pages/_app.js]: should be using js-cookie (or even bettter cookie-next)

File: package.json [hunk 5]
```diff
@@ -14,10 +14,14 @@
     "@svgr/webpack": "^5.5.0",
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
+    "i18next": "^23.2.0",
+    "js-cookie": "^3.0.5",
```
Comment (robdll) [inline package.json]: please use [cookie-next](https://www.npmjs.com/package/cookies-next)

File: pages/index.js [hunk 6]
```diff
@@ -6,8 +6,12 @@ import Services from "./components/Services";
 import Testimonial from "./components/Testimonial";
 import Subscribe from "./components/Subscribe";
 import Footer from "./components/Footer";
-
+import { useTranslation } from "react-i18next";
+import { useEffect } from "react";
```
Comment (robdll) [inline pages/index.js]: useEffect doesn't seems to be used

File: pages/index.js [hunk 7]
```diff
@@ -6,8 +6,12 @@ import Services from "./components/Services";
...
 export default function Home() {
+  const { t } = useTranslation();
+
```
Comment (robdll) [inline pages/index.js]: remove empty lines

File: pages/services/index.js [hunk 8]
```diff
@@ -14,10 +15,10 @@ export default function Home() {
...
       <Subscribe></Subscribe>
       <Footer></Footer>
-      <div className="footer__rights"> 
-        Todos os direitos reservados por Cintya Flores
-      <br />
-        Meu Arteliê 2021
+      <div className="footer__rights">
+        {t("footerTermsTitle")}
+        <br />
+        {t("footerTermsSubtitle")}
```
Comment (robdll) [inline pages/services/index.js]: {t("footerTermsSubtitle") + new Date().getFullYear()}

File: pages/components/Testimonial.js [hunk 9]
```diff
@@ -1,74 +1,71 @@
...
+      <div className={styles.testimonial__photo_container}>
+        <Image
+          className={styles.testimonial__photo}
+          src={props.item.imageUrl}
+          alt="Testimonial"
+          width={312}
+          height={300}
+        />
+      </div>
+      <p className={styles.testimonial__copy}>{props.item.feedback}</p>
```
Comment (robdll) [inline pages/components/Testimonial.js:64]: feedbacks should be moved to the translation file, and this should be using translation

File: pages/components/MenuOverlay.js [hunk 10]
```diff
@@ -1,29 +1,72 @@
...
+            <button
+              data-id={`${language.code}-button`}
+              // className={i18n.language === language.code ? "active" : undefined}
+              onClick={() => changeLanguage(i18n, language.code, router)}
+              key={language.code}
+            >
+              {t(language.translateKey)}
+            </button>
+          ))}
+        </div>
```
Comment (robdll) [inline pages/components/MenuOverlay.js]: no UI changes please

File: pages/components/MenuOverlay.js [hunk 11]
```diff
@@ -1,29 +1,72 @@
...
-import { useRouter } from 'next/router'
+const changeLanguage = (i18n, language) => {
+  window.document.cookie = "MY_LANGUAGE="+language
+  i18n.changeLanguage(language);
+};
+
+const languages = [
+  { code: "pt", translateKey: "Portuguese" },
+  { code: "en", translateKey: "english" },
+];
```
Comment (robdll) [inline pages/components/MenuOverlay.js]: remove this, for now we just want the configuration

File: public/locales/en/translation.json [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+    "serviceVideoContent": " The teaching method was developed by me to work with the greatest convenience for your routine in times of pandemic and social isolation. There is a huge amount of classes divided into blocks of 4 videos that will be assembled according to your interest, need or technical level. Each class has a challenge to be fulfilled and only after completing the challenge can the next video be accessed.",
+    "serviceVideoSubContent": " I monitor the entire process via whatsapp, giving suggestions and support in its evolution. The classes are designed to teach shapes and effects with colored pencils and some other materials (if you are interested) with the aim of helping to improve the artist's own style, so that his style is always unique!",
+    "serviceHeader3": "Consultation",
+    "serviceHeader4": "Workshop",
+    "contactTitle": "Contact-Me",
+    "contactSubTitle1": "Your name",
+    "contactSubTitle2": "Enter email address",
+    "contactSubTitle3": "Your Text Here",
+    "contactButtonSend": "To send",
+    "contactButtonSent": "Sent"
```
Comment (robdll) [inline public/locales/en/translation.json]: in general, you can simplify the translation files maintenance by providing namespace (nesting objects).
for example, you could use 

```
"contact": {
   "title": "Contact-Me",
   "placeholders" {
       "name": "Your name",
        "email": "Enter email address",
        "message": "Your Text Here",
      }
}

and access them as
```
    t('contact.title') 
    t('contact.placeholders.name') 
    
```

**Review by robdll — COMMENTED** [review]
Left some comment, let me know if you can handle them

**Comment by tarunsengar1987** [comment]
Please review the code as per comment everypart is updated

**Pull request closed without merging**